src/Resources/views/BlockAdmin/compose_preview.html.twig

```diff
@@ -7,63 +7,75 @@
     <a class="page-composer__container__child__edit"
        href="{{ blockAdmin.generateUrl('edit', { 'id': child.id, 'composer': true }) }}"
     >
-        {% set service = attribute(blockServices, child.type) %}
-        {% if service.metadata is defined %}
-            {% set metadata = service.metadata %}
+        {% set service = attribute(blockServices, child.type) ?? null %}
+        {% if service is not null %}
+            {% if service.metadata is defined %}
+                {% set metadata = service.metadata %}
+            {% else %}
+                {% set metadata = service.blockMetadata %}
+            {% endif %}
+            <h4 class="page-composer__container__child__name">
+                {{ child.name|default(metadata.title)|trans({}, metadata.domain|default('SonataPageBundle')) }}
+            </h4>
+            {% if not metadata.image %}
+                <i class="{{ metadata.option('class') }}" ></i>
+            {% else %}
+                <img src="{{ asset(metadata.image) }}" style="max-height: 20px; max-width: 100px;"/>
+            {% endif %}
+            <small>{{ metadata.title|trans({}, metadata.domain|default('SonataPageBundle')) }}</small>
+            <span class="page-composer__container__child__toggle">
+                <i class="fa fa-chevron-down"></i>
+                <i class="fa fa-chevron-up"></i>
+            </span>
         {% else %}
-            {% set metadata = service.blockMetadata %}
+            {{ child.type }}
```

Comment:
@JeromeEngelnAdeliom  could you just add a red Background for blocks that doesn't exist and "maybe a message" bellow the block type (block not found)